vendordeps/photonlib.json

```diff
@@ -0,0 +1,57 @@
+{
+    "fileName": "photonlib.json",
+    "name": "photonlib",
+    "version": "v2024.2.4",
```

Comment:
FYI that latest version is v2024.2.6

Reply:
Looks like [v2024.2.5](https://github.com/PhotonVision/photonvision/releases/tag/v2024.2.5) and [v2024.2.6](https://github.com/PhotonVision/photonvision/releases/tag/v2024.2.6) are very new, and v2024.2.6 has substantial changes that will take time to investigate (without a lot of added functionality). We'll probably do it later in the season, but I'd like to get this in relatively soon.